aospy/test/test_roundtrip.py

```diff
@@ -199,52 +220,104 @@ def test_vert_int_pressure(self):
         pass
 
 
+@unittest.skipIf(not model_files_exist(idealized_moist_rad), skip_message)
+class TestIdealizedMoistRad(TestAM2):
+    def setUp(self):
+        self.two_d_test_params = {'proj': aospy_test,
+                                  'model': idealized_moist_rad,
+                                  'run': test_idealized_moist_rad,
+                                  'var': t_surf,
+                                  'date_range': ('0003-01-01', '0006-12-31'),
+                                  'intvl_in': 'monthly',
+                                  'dtype_in_time': 'ts',
+                                  'dtype_out_vert': False,
+                                  'level': False}
+        self.three_d_test_params = {'proj': aospy_test,
+                                    'model': idealized_moist_rad,
+                                    'run': test_idealized_moist_rad,
+                                    'var': temp,
+                                    'date_range': ('0003-01-01',
+                                                   '0006-12-31'),
+                                    'intvl_in': 'monthly',
+                                    'dtype_in_time': 'ts',
+                                    'level': False}
+        self.sub_daily_test_params = {'proj': aospy_test,
+                                      'model': idealized_moist_rad,
+                                      'run': test_idealized_moist_rad,
+                                      'var': temp,
+                                      'date_range': ('0003-01-01',
+                                                     '0006-12-31'),
+                                      'intvl_in': '3-hourly',
+                                      'dtype_in_time': 'inst',
+                                      'level': False}
+
+    @unittest.skip('not tested in idealized moist rad model')
```

Comment:
As an aside, the fact that we have all these skips for the idealized moist case means that our overall structure should be improved...there should be separate mix-ins for the tests applicable to all models and for those only applicable to the full physics one.  (That was my mistake)

Doesn't need to be addressed in this PR though, nor is it that important, just thinking out loud